### ISSUE TITLE
Port TensorFlow-based code in DeepChem to PyTorch

### ISSUE THREAD

**Issue by ShigrafS** [deepchem/deepchem#4304]
**Description:**

Hello DeepChem team,

I would like to propose an enhancement to port the following TensorFlow-based code to PyTorch to improve compatibility with PyTorch users and provide greater flexibility for those working with this framework. Here are the files and lines that need to be addressed:

- `deepchem/rl/ppo.py`: lines 259, 388, 569
- `deepchem/models/normalizing_flows.py`: line 200
- `deepchem/models/keras_model.py`: lines 512, 718
- `deepchem/rl/a2c.py`: lines 361, 480

These files are currently implemented using TensorFlow, and it would be beneficial to port them to PyTorch.

**Comment by ShigrafS** [comment]
@rbharath I am working on this.

**Comment by rbharath** [comment]
I believe these have all been ported now. Closing this issue
